jmx-metrics/src/main/resources/target-systems/activemq.groovy

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+def activemqMetrics = otel.mbeans(
+  [
+    "org.apache.activemq:type=Broker,brokerName=*,destinationType=Queue,destinationName=*",
+    "org.apache.activemq:type=Broker,brokerName=*,destinationType=Topic,destinationName=*"
+  ]
+)
+
+
+otel.instrument(activemqMetrics,
+  "activemq.producer.count",
+  "The number of producers currently attached to the broker.",
+  "{producers}",
+  ["destination" : { mbean -> mbean.name().getKeyProperty("destinationName") }],
```

Comment:
Can we also set the `brokerName` as a label/attribute? I think it's applicable for all metrics here.

Reply:
sure thing